### ISSUE TITLE
Uncaught SyntaxError: Invalid regular expression: /(/: Unterminated group

### ISSUE THREAD

**Issue by samdoiron** [nikhilkalige/docblockr#46]
Create a new documentation block just above an Annotation in java
Ex.

```
/**<cr>
@Override
public boolean isEmpty() {
    // snip
}


**Atom Version**: 0.177.0
**System**: Mac OS X 10.10
**Thrown From**: [docblockr](https://github.com/nikhilkalige/docblockr) package, v0.5.8

### Stack Trace

Uncaught SyntaxError: Invalid regular expression: /(/: Unterminated group

```
At /Users/samdoiron/.dotfiles/atom.symlink/packages/docblockr/lib/docsparser.js:1111

SyntaxError: Invalid regular expression: /(/: Unterminated group
  at new RegExp (native)
  at RegExp (native)
  at String.search (native)
  at DocsParser.JavaParser.get_definition (/Users/samdoiron/.dotfiles/atom.symlink/packages/docblockr/lib/docsparser.js:1111:26)
  at DocBlockrAtom.module.exports.DocBlockrAtom.initialize (/Users/samdoiron/.dotfiles/atom.symlink/packages/docblockr/lib/docblockr-worker.js:704:26)
  at DocBlockrAtom.module.exports.DocBlockrAtom.parse_command (/Users/samdoiron/.dotfiles/atom.symlink/packages/docblockr/lib/docblockr-worker.js:258:12)
  at atom-workspace.<anonymous> (/Users/samdoiron/.dotfiles/atom.symlink/packages/docblockr/lib/docblockr-worker.js:64:16)
  at atom-workspace.handler (/Applications/Atom.app/Contents/Resources/app/src/space-pen-extensions.js:115:32)
  at atom-workspace.jQuery.event.dispatch (/Applications/Atom.app/Contents/Resources/app/node_modules/space-pen/vendor/jquery.js:4681:9)
  at atom-workspace.elemData.handle (/Applications/Atom.app/Contents/Resources/app/node_modules/space-pen/vendor/jquery.js:4359:46)
  at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app/src/command-registry.js:243:29)
  at /Applications/Atom.app/Contents/Resources/app/src/command-registry.js:3:61
  at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (/Applications/Atom.app/Contents/Resources/app/node_modules/atom-keymap/lib/keymap-manager.js:558:16)
  at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/Applications/Atom.app/Contents/Resources/app/node_modules/atom-keymap/lib/keymap-manager.js:396:22)
  at HTMLDocument.module.exports.WindowEventHandler.onKeydown (/Applications/Atom.app/Contents/Resources/app/src/window-event-handler.js:170:20)

```

### Commands

```
     -0:10.4 core:save (atom-text-editor.editor.vim-mode.autocomplete-active.is-focused.insert-mode)
     -0:10.3 vim-mode:move-up (atom-text-editor.editor.vim-mode.autocomplete-active.is-focused.insert-mode)
  7x -0:10.2 vim-mode:move-down (atom-text-editor.editor.vim-mode.autocomplete-active.is-focused.insert-mode)
     -0:09.1 vim-mode:move-up (atom-text-editor.editor.vim-mode.autocomplete-active.is-focused.insert-mode)
  3x -0:08.7 vim-mode:move-down (atom-text-editor.editor.vim-mode.autocomplete-active.is-focused.insert-mode)
  4x -0:07.8 vim-mode:move-up (atom-text-editor.editor.vim-mode.autocomplete-active.is-focused.insert-mode)
  3x -0:07.1 vim-mode:move-down (atom-text-editor.editor.vim-mode.autocomplete-active.is-focused.insert-mode)
  7x -0:06.3 vim-mode:move-up (atom-text-editor.editor.vim-mode.autocomplete-active.is-focused.insert-mode)
 10x -0:04.0 vim-mode:move-down (atom-text-editor.editor.vim-mode.autocomplete-active.is-focused.insert-mode)
     -0:02.9 vim-mode:move-up (atom-text-editor.editor.vim-mode.autocomplete-active.is-focused.insert-mode)
     -0:02.8 vim-mode:insert-below-with-newline (atom-text-editor.editor.vim-mode.autocomplete-active.is-focused.insert-mode)
     -0:02.2 docblockr:parse-tab (atom-text-editor.editor.vim-mode.autocomplete-active.is-focused.insert-mode)
     -0:02.2 snippets:next-tab-stop (atom-text-editor.editor.vim-mode.autocomplete-active.is-focused.insert-mode)
     -0:02.2 snippets:expand (atom-text-editor.editor.vim-mode.autocomplete-active.is-focused.insert-mode)
     -0:02.2 editor:indent (atom-text-editor.editor.vim-mode.autocomplete-active.is-focused.insert-mode)
     -0:00.7 docblockr:parse-enter (atom-text-editor.editor.vim-mode.autocomplete-active.is-focused.insert-mode)
```

### Config

```json
{
  "core": {
    "ignoredNames": [
      ".bundle",
      ".git",
      "log",
      "repositories",
      "tmp",
      "vendor"
    ],
    "disabledPackages": [
      "background-tips",
      "exception-reporting",
      "metrics",
      "linter-javac"
    ],
    "themes": [
      "isotope-ui",
      "solarized-dark-syntax"
    ]
  }
}
```

### Installed Packages

```coffee
# User
atom-color-highlight, v3.0.6
autocomplete-plus, v2.1.3
docblockr, v0.5.8
isotope-ui, v1.2.10
linter, v0.11.1
linter-clang, v2.25.1
minimap, v4.1.2
vim-mode, v0.30.0

# Dev
No dev packages
```

/cc @atom/core

**Comment by BusyJay** [comment]
Is it OK to auto create an issue without use's permission? Is it Atom's or docblockr's behavior?

**Comment by MoritzKn** [comment]
This was fixed with #46. In version 0.8.0 it shouldn't happen anymore.
